Sources/TuistEnvKit/Commands/CommandRunner.swift

```diff
@@ -64,9 +64,9 @@ class CommandRunner: CommandRunning {
 
         switch resolvedVersion {
         case let .bin(path):
-            logger.notice("Using bundled version at path \(path.pathString)")
+            logger.debug("Using bundled version at path \(path.pathString)")
         case let .versionFile(path, value):
-            logger.notice("Using version \(value) defined at \(path.pathString)")
+            logger.debug("Using version \(value) defined at \(path.pathString)")
```

Comment:
Does this mean you wont see this info unless you run `--verbose`?

Reply:
correct